packages/require/lib/index.ts

```diff
@@ -0,0 +1,130 @@
+import path from "path";
+import TruffleConfig from "@truffle/config";
+import { options as expectOptions } from "@truffle/expect";
+import { Web3Shim, createInterfaceAdapter } from "@truffle/interface-adapter";
+
+import vm from "vm";
+import Module from "module";
+
+import originalRequire from "original-require";
```

Comment:
This is a holdover from the way that we were doing this before. Referencing `require` this way allows us to mark this module as external in our WebPack config, which gives us a reference to NodeJS's builtin `require` rather than WebPack's one.

Reply:
Will this assignment work as desired if you do that? Why wouldn't it still use webpack's `require`?

Reply:
Well, depending on where you to it you might have to do shenanigans like `(0, eval)("require")`. But the reason is works is because it gets `require` from the global scope, which belongs do node.

Reply:
Oh so it kind of "bypasses" webpack's `require`?

Reply:
right

Reply:
`eval("require")` works fine, but I get weird results with `global.require`. I see that `original-require` is used in a number of Truffle packages however, so I think I’ll leave removal of that package for another time